packages/share_plus_web/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+# 0.0.1
```

Comment:
Can we start with version 0.1.0 to avoid minus points on pub.dev? The platform interface API is not that unstable that we need to start with 0.0.1, I think. At least we have started at 0.1.0 with the others. :smile:

Reply:
made it 0.1.0